modules/massrole/commands/massrole.js

```diff
@@ -0,0 +1,202 @@
+const {localize} = require('../../../src/functions/localize');
+let target;
+
+module.exports.subcommands = {
+    'add': async function (interaction) {
+        await checkTarget(interaction);
+        if (target === 'all') {
+            await interaction.deferReply({ ephemeral: true });
+            interaction.guild.members.cache.forEach(member => {
+                member.roles.add(interaction.options.getRole('role'));
+            });
```

Comment:
For each does not return a promise and will send the "done"-reply way to early
```suggestion
    for(const member of interaction.guild.members.cache.values()) {
          await member.roles.add(interaction.options.getRole('role'));
    }
```